xml/security_ldap_sssd.xml

```diff
@@ -55,10 +55,15 @@
   </step>
   <step>
    <para>
-     Create your new SSSD configuration. The following example creates a
-     basic client configuration:
+     Create your new SSSD configuration template. The allowed output file
+     names are <literal>sssd.conf</literal> and
+     <literal>ldap.conf</literal>.
+     <literal>display</literal> sends the output to stdout. The following
+     example creates a client configuration in
+     <filename>/etc/sssd/sssd.conf</filename>:
    </para>
-   <screen>&prompt.sudo;<command>sh -c 'dsidm <replaceable>INSTANCE_NAME</replaceable> client_config > /etc/sssd/sssd.conf'</command>
+   <screen>&prompt.sudo;<command>sudo cd /etc/sssd</command>
+&prompt.sudo;<command>dsidm <replaceable>LDAP1</replaceable> client_config sssd.conf</command>
```

Comment:
Not a good idea to just yolo this into /etc/sssd/sssd.conf. Better to put it into /root/ and then move it. Also remember, that the permissions on /etc/sssd/sssd.conf are very finincky so you need to set them correctly too.

Reply:
Line 52, make a backup of the existing sssd.conf first. Permissions are addressed on line 117. Does that look OK?

Reply:
Then it needs some examples of how to test the new setup, and some SSH and TLS stuff, which will be in a separate PR.